Formula/safe.rb

```diff
@@ -0,0 +1,40 @@
+class Safe < Formula
+  desc "Vault CLI"
+  homepage "https://github.com/starkandwayne/safe"
+  url "https://github.com/starkandwayne/safe/archive/v0.8.1.tar.gz"
+  sha256 "cc766996cfff2109c27f98be0430fbc0d16076cb6772de6b883307e9d95b6aae"
+
+  depends_on "go" => :build
+  depends_on "gox" => :build
+  depends_on "vault"
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["RELEASE_ROOT"] = buildpath/"destdir"
+    ENV["TARGETS"] = "darwin/amd64"
+    ENV["VERSION"] = version
+
+    (buildpath/"src/github.com/starkandwayne/safe").install buildpath.children
+
+    (buildpath/"destdir").mkpath
+
+    cd "src/github.com/starkandwayne/safe" do
+      system "make", "build"
```

Comment:
What are we gaining here by doing it this way instead of just `go build -ldflags="-X main.Version=#{version}"`, etc? I'm not really sure what `gox` is helping us with here given we've already built the binary by this point.

Reply:
what it potentially gains us is future-proofing.  the upstream developer clearly wants to use `make` as a build tool; if we use that same abstraction, then hopefully when at some future point in time he changes how the software is built, we'll still be able to successfully build by running `make build`.

Reply:
Well, I guess we could still use `make build`. Just confused as to what `gox` is being used for after the binary has already been built. I've not spent much time with `gox` itself so maybe I'm missing something here.